src/link_representation.ts

```diff
@@ -59,9 +59,9 @@ export function createGroupRep(context: IBandContext, a: IVisWrapper, aa: Rect,
         loc : locs[i] ? locs[i].aabb() : null
       }; };
     }
-    var groupa = data[0].groupa.map(more(toArray(data[1])));
-    var groupb = data[0].groupb.map(more(toArray(data[2])));
-    var r = [];
+    const groupa = (<any>data[0]).groupa.map(more(toArray(data[1])));
+    const groupb = (<any>data[0]).groupb.map(more(toArray(data[2])));
```

Comment:
Try to checkout all tslint fix branches and your branch. At the end it worked in my local environment.